api/src/beacons/dto/find-beacons.dto.ts

```diff
@@ -16,4 +16,8 @@ export class FindBeaconsDto {
   @IsNotEmpty()
   @IsNumber()
   maxDistance: number;
+
+  @IsOptional()
+  @IsArray()
```

Comment:
I think there's a way to go a bit deeper and not only validate that it's an array, but that it's an array of strings.

Reply:
Something kinda like this, but please double check.
`@IsArray()
  @IsString({ each: true })
  @ValidateNested({ each: true })`